src/lizaalert/courses/admin.py

```diff
@@ -45,6 +47,23 @@ class ChapterInline(admin.TabularInline):
     model = Chapter
     min_num = 1
     extra = 0
+    readonly_fields = ("get_chapter_link",)
+    # Устанавливаем порядок отображения полей
+    fields = (
+        "get_chapter_link",
+        "deleted_at",
+        "order_number",
+        "title",
+        "user_created",
+        "user_modifier",
+    )
+
+    # Метод для отображения ссылки на главу курса
+    def get_chapter_link(self, obj):
+        url = reverse("admin:courses_chapter_change", args=(obj.id,))
+        return format_html('<a href="{}">{}</a>', url, obj.title)
+
+    get_chapter_link.short_description = "Ссылка"
```

Comment:
по сути этот объект является главой

```suggestion
    get_chapter_link.short_description = "Глава"
```